package.json

```diff
@@ -34,7 +34,7 @@
   },
   "scripts": {
     "clear": "rm -Rf tmp",
-    "test": "npm run eslint && cross-env NODE_ENV=test nyc --reporter=lcovonly --reporter=text-summary npm run mocha",
+    "test": "npm run eslint && cross-env NODE_ENV=test nyc --reporter=lcovonly --reporter=text-summary npm run mocha && npm run cucumber",
     "test:mongo": "npm run eslint && cross-env NODE_ENV=test nyc --reporter=lcovonly --reporter=text-summary npm run mocha:mongo",
     "eslint": "eslint server/. test/. --config .eslintrc.json",
     "start": "concurrently 'mongod' 'wait-on tcp:27017 && cross-env NODE_ENV=production node server/'",
```

Comment:
In the script `dev:debug` the `NODE_ENV=development` is hard coded. If you want to run the backend separately from cucumber tests (for debugging) make sure you change `NODE_ENV=test` so both the cucumber test and your backend running in the background are using the `test` database.